data/all_locations.json

```diff
@@ -32244,6 +32244,7 @@
                 "latitude": "30.40116159999999",
                 "longitude": "-86.9844129",
                 "id": 3311,
+                "multimachine": 2,
```

Comment:
why is that in the all_locations.json? was that on purpose?

Reply:
Yes, there was a glitch before. During this PR I had first added the machine manually and then it was later added by the bot. So the same entry appeared twice in server_locations which threw an error. Not sure why the bot did not modify the existing entrance. 

The `multimachine` I had read it from a user comment, so the bot couldnt pick it up so I decided to add it manually here